libcloudforensics/providers/azure/internal/network.py

```diff
@@ -87,7 +87,7 @@ def CreateNetworkInterface(self,
       # NIC doesn't exist, ignore the error and create it
 
     # pylint: disable=unbalanced-tuple-unpacking
-    public_ip, _, subnet = self._CreateNetworkInterfaceElements(
+    public_ip, _, subnet, nsg = self._CreateNetworkInterfaceElements(
```

Comment:
nit: maybe make the name more explicit, ie netsecuritygrp ?

Reply:
`NSG` is the official Azure abbreviation so if possible I'd rather stick with this, but I added a comment above that line (and the line for the next comment) to make it a little clearer. Let me know what you think!

Reply:
SGTM, thanks!